cip/1.accepted/CIP2016-06-14-Define-comparability-and-equality-as-well-as-orderability-and-equivalence.adoc

```diff
@@ -118,7 +118,7 @@ We propose that <<comparability-con,comparability>> should be defined between an
   * Strings are <<incomparable>> to any value that is not also a string.
 - Lists
   * Lists are compared in dictionary order, i.e. list elements are compared pairwise in ascending order from the start of the list to the end. Elements missing in a shorter list are considered to be less than any other value (including `null` values).
-  For example, `[1] < [1, 0]` but also `[1] \<= [1, null]`.
+  For example, `[1] < [1, 0]` but also `[1] \< [1, null]`.
```

Comment:
Aha, well the issue I'm having is that I think the example clashes with the sentence before, where absent elements are claimed to be less that `null`. Then `[1] <> [1,null]`, which should also match the notion of the unknown null, meaning that the lists cannot be identical.